### ISSUE TITLE
what's the meaning of test set, doesn't only dev set is okay?

### ISSUE THREAD

**Issue by zhangpengGenedock** [tensorflow/nmt#124]
what's the meaning of test set, doesn't only dev set is okay?
can someone explantation it? I will be very appreciate.

**Comment by oahziur** [comment]
In general, we only want to use the dev set to tweak our model, and we
select the model that performs best on the dev set as our best model.

We use the test set to double check our best model, and report the
performance. In this way, we can avoid selecting and reporting a model that
just overfits on the test set.

For example, you can set test_prefix to None when you doing hyper
parameters search.

**Comment by zhangpengGenedock** [comment]
@oahziur thank you !
